.github/renovate.json5

```diff
@@ -0,0 +1,28 @@
+{
+	"extends": [
+		"config:base",
+		"schedule:weekly", // each monday
+		"group:allNonMajor", // batch non-majors into one PR
+		":semanticCommits" //nicer commit messages
+	],
+	"labels": ["dependencies"],
+	"ignorePaths": [
+		"packages/playground/**",
```

Comment:
I think updating playground deps would be nice too, so we don't have to manually do so 🤔

Reply:
agreed. removed the whole ignorePaths as we don't have tests with fixed dep versions